src/styles/feedback.scss

```diff
@@ -13,7 +10,7 @@
   padding-left: 0;
```

Comment:
`text-wrap` is unsupported in FF (and not great support other places) can we use `white-space: nowrap` instead?